frontend/test/resolution/testTypePropertyPrimitives.cpp

```diff
@@ -347,7 +348,7 @@ static void test5() {
 static void test6() {
   Test tpg {
     .testName=__FUNCTION__,
-    .isChplHomeRequired = true,
+    .isChplHomeRequired= true,
```

Comment:
Was this an intentional change?

Reply:
Yes and no -- it made the regex easier, but also, it's consistent with the rest of the fields in this comment now.